scripts/ddp-RxTerms-script/src/index.ts

```diff
@@ -1,33 +1,89 @@
 import * as XLSX from "xlsx";
 import { RxNormDrugOption } from "./models";
-import * as fs from 'fs';
+import * as fs from "fs";
 
 const inputFileName = process.argv[2] || "RxTerms202202.txt";
 const outputFileName = process.argv[3] || "output.txt";
+const MAX_LENGTH = 45;
 
-const translations: string[] =  process.argv.filter(arg => ['en', 'es', 'ru'].includes(arg));
+const groupBy = <T, K>(list: T[], getKey: (item: T) => K) => {
+  const map = new Map<K, T[]>();
+  list.forEach((item) => {
+    const key = getKey(item);
+    const collection = map.get(key);
+    if (!collection) {
+      map.set(key, [item]);
+    } else {
+      collection.push(item);
+    }
+  });
+  return Array.from(map.values());
+};
+
+const translations: string[] = process.argv.filter((arg) =>
+  ["en", "es", "ru"].includes(arg)
+);
 
 const workBook = XLSX.readFile(inputFileName);
 
 const workSheet = workBook.Sheets[workBook.SheetNames[0]];
 
 const rxNormArray = XLSX.utils.sheet_to_json(workSheet) as RxNormDrugOption[];
 
-const picklistOptions = rxNormArray.map((drug) => ({
-  stableId: drug.RXCUI.toString(),
+const filteredRxNormsArray = groupBy(rxNormArray, (x) => x.DISPLAY_NAME).map(
+  (group) => group[0]
+);
+
+const picklistOptions = filteredRxNormsArray.map((drug) => ({
+  stableId: generateStableID(drug.DISPLAY_NAME),
```

Comment:
I think you are going to have to check that the generated stable id is really unique/cannot count on the current transformation giving you a unique ids.